src/main/java/com/seiama/javaducks/controller/JavadocController.java

```diff
@@ -91,6 +94,18 @@ public ResponseEntity<?> redirectToPathWithTrailingSlash(
       .build();
   }
 
+  @GetMapping("/{project:[a-z]+}")
```

Comment:
this only makes https://jd.papermc.io/paper work, not https://jd.papermc.io/paper/

we should add another mapping with the trailing slash that calls the same method (similarly to how `redirectToPathWithTrailingSlash` and `serveJavadocs` are basically the same mapping minus the trailing slash)

I would also move the method up or down so that you don't split the two mappings with the version